packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -282,15 +268,15 @@ let create: <
  * including functionality that does not have to be kept consistent across versions or deterministic.
  *
  * This can include policy for how to use this schema for "view" purposes, and well as how to expose editing APIs.
- * Use {@link makeFieldSchema} to create an instance of this type instead.
+ * Use `createFieldSchema` to create an instance of this type instead.
```

Comment:
This comment is public facing, so it should not tell you to use non public APIs.
```suggestion
 * Use {@link SchemaFactory} to create the FieldSchema instances, for example {@link SchemaFactory.optional}.
 * @privateRemarks
 * Public access to the constructor is removed to prevent creating expressible but unsupported (or not stable) configurations.
 * {@link createFieldSchema} can be used internally to create instances.
```

Reply:
Updated docstring to suggested change